src/comm.c

```diff
@@ -132,6 +132,7 @@ fd_set out_set;   /* Set of desc's for writing  */
 fd_set exc_set;   /* Set of desc's with errors  */
 int maxdesc;
 char *alarm_section = "(unknown)";
+bool MOBtrigger;

```

Comment:
Since mud.h had ''''bool MOBtrigger;''' declared, it was initialized every time the file was imported somewhere.  As a global it should be declared as an extern and then declared as a global in a centralized .c file.